### DISCUSSION TITLE
SSO Google missing /

### DISCUSSION THREAD

**Discussion by BinhPhuongIT** [nocodb/nocodb#3596] [Q&A]
![image](https://user-images.githubusercontent.com/69758987/189815519-69202901-c55e-4209-a596-111dbc7276e7.png)

**Comment by BinhPhuongIT** [comment]
https:/nocodb.com/dashboard  --->>https://nocodb.com/dashboard

**Reply by o1lab** [reply]
@pranavxc

**Reply by BinhPhuongIT** [reply]
I am using version 0.96.2.
I tried ver 0.92.4 but still got the above error.

**Reply by o1lab** [reply]
Ahok, if it isnt working in 0.96.2 - you would have to see why coz there are others who are using it fine.
